learning_resources_search/plugins.py

```diff
@@ -18,11 +34,7 @@ def resource_upserted(self, resource):
         Args:
             resource(LearningResource): The Learning Resource that was upserted
         """
-        if resource.resource_type == COURSE_TYPE:
-            search_index_helpers.upsert_course(resource.id)
```

Comment:
I removed search_index_helpers, since the purpose of search index helpers was to be an interface between the loaders and the methods in tasks. Now the methods in plugins serve the same purpose.

This should make the code a bit less confusing since previously we would had methods with the same name in search_index_helpers, tasks, and indexing_api and that made searching for method names in the code and following the logic harder.